src/Datadog.Trace/SamplingPriority.cs

```diff
@@ -0,0 +1,29 @@
+namespace Datadog.Trace
+{
+    /// <summary>
+    /// Valid values for <see cref="SpanContext.SamplingPriority"/>.
+    /// These values determine whether a trace should be sampled or not.
+    /// </summary>
+    public enum SamplingPriority
+    {
+        /// <summary>
+        /// Explicitly ask the backend to not store a trace.
+        /// </summary>
+        UserReject = -1,
+
+        /// <summary>
+        /// Used by the built-in sampler to inform the backend that a trace should be rejected and not stored.
+        /// </summary>
+        AutoReject = 0,
+
+        /// <summary>
+        /// Used by the built-in sampler to inform the backend that a trace should be kept and stored.
+        /// </summary>
+        AutoKeep = 1,
+
+        /// <summary>
+        /// Explicitly ask the backend to keep a trace.
+        /// </summary>
+        UserKeep = 2
+    }
+}
```

Comment:
Missing newline at end of file here.

Reply:
Or maybe this file is generated by some dotnet magic?

Reply:
Seems like something our linter should catch. Fixed in 02af7fcc48b4ac33a45f108dc706c0877c6d64d5.